styles/hud/hud.scss

```diff
@@ -58,7 +58,6 @@ Hud,
 #HudCenter {
 	width: 100%;
 	height: 100%;
-	margin-top: 300px;
```

Comment:
Doesn't this affect other HUD components?

Reply:
Nothing else is using that style. See [layout/hud/hud.xml](https://github.com/momentum-mod/panorama/pull/204/files#diff-18957eb64f292089f2fc9c46c1acca5a601af40648f15df8351e37ff42d63356)